tests/test_base.py

```diff
@@ -0,0 +1,54 @@
+from api import models
+from django.test import Client
+from django.contrib.auth import login
+from django.test import TestCase, Client
+from rest_framework.test import force_authenticate, APIRequestFactory, APIClient
+from django.urls import reverse
+from .factories import UserFactory, ItemFactory, GroupFactory, CollectionFactory, MeetingFactory
+
+
+class SimpleTest(TestCase):
+    meetings_url = '/collection/1'
+
+    def setUp(self):
+
+        # All users have password "password123" by default
+        # Username defaults to first_initial+last_name (i.e. jdoe)
+        self.owner = UserFactory(username="owner")
+        self.owner.save()
+
+        self.random_user = UserFactory(username="random_user")
```

Comment:
Since`random_user` isn't being used anywhere would it make sense to take it out for now?